### PR TITLE
Adds Knight Arena to Holodeck

### PR DESCRIPTION
**Pull request by Glorken**
Creates a holodeck option akin to the thunderdome, but with knight armor
and claymores. Also adds blue and red claymore sprites for the teams.

Also changes the part that makes emagged holo eswords do real damage from weapon/holo/esword to weapon/holo to make it apply to claymores. Please tell me if this was a bad idea.

First real time with github and pull requests, so please be gentle. Also, please tell me if I mapmerged wrong.

Pics:
Arena
knightarena
Windows disappear as usual
scrnshot5
Non-lethal with safety
scrnshot6
Lethal without
scrnshot7
Modelling the blue claymore
scrnshot1scrnshot2
scrnshot3scrnshot4

:cl: Glorken
rscadd: Added a Knight Arena for Holodeck.
imageadd: Added red and blue claymore sprites.
spellcheck: Changed overrided to overrode when emagging Holodeck computer.
/:cl:

I have tested the following successfully:
- [x] Loading Knight Arena
- [x] Removing divider by using the buttons
- [x] Knocking a person out with both holo-claymore and holo-esword without brute damage
- [x] Emagging the console and killing with both holo-claymore and holo-esword
- [x] Removal of holo-items on exit/switch
- [x] Loading every holo area

### CODE REVIEW

**Comment by Gummin** [comment]
Real damage = probably a really bad idea

**Comment by Plotron** [comment]
We absolutely can't afford real damage there.

**Comment by Plotron** [comment]
Lethal claymores? Sure, why not?

**Comment by Glorken** [comment]
Which part did I mess up? If you're talking about what I said in the PR, I meant the loop that happens when an emagged holo computer loads a place with holo eswords. It changes them from doing Stalin's damage to brute damage.

File: code/game/machinery/computer/HolodeckControl.dm [hunk 1]
```diff
@@ -290,7 +296,7 @@
 	holographic_items = A.copy_contents_to(linkedholodeck , 1)
 
 	if(emagged)
-		for(var/obj/item/weapon/holo/esword/H in linkedholodeck)
+		for(var/obj/item/weapon/holo/H in linkedholodeck)
```
Comment (Glorken) [inline code/game/machinery/computer/HolodeckControl.dm]: Specifically, this line.

**Comment by Glorken** [comment]
Accident

**Comment by NTSAM** [comment]
:+1: :heavy_multiplication_x: :100: 
Time fer some medieval duels

**Comment by Fox-McCloud** [comment]
I recommend very thorough testing of this; the holodeck is finicky as hell.

**Comment by Earthdivine** [comment]
>Holodeck
>Finicky

Your giving it too much credit, @Fox-McCloud

**Comment by Fox-McCloud** [comment]
Also, pics.

**Comment by Glorken** [comment]
Will do later tomorrow.

**Comment by Glorken** [comment]
@Fox-McCloud 
Pics added

**Pull request merged**